packages/vite/src/node/server/moduleGraph.ts

```diff
@@ -27,7 +27,7 @@ export class ModuleNode {
   importers = new Set<ModuleNode>()
   importedModules = new Set<ModuleNode>()
   acceptedHmrDeps = new Set<ModuleNode>()
-  isSelfAccepting = false
+  isSelfAccepting: boolean | null = null
```

Comment:
Conceptually, I would prefer this one to be:
```suggestion
  isSelfAccepting?: boolean
```
As the value is undefined until we compute it. In the case of `transformResult`, I think is different because the value is reset to null.

Reply:
Makes sense. Done.